### PR TITLE
updated main dependencies

### PR DESCRIPTION
**Pull request by cxoltero**

### CODE REVIEW

**Comment by obuckley-kenzan** [comment]
Can one of the admins verify this patch?

File: package.json [hunk 1]
```diff
@@ -42,9 +42,9 @@
   "license": "Apache-2.0",
   "dependencies": {
     "gulp": "3.9.1",
-    "gulp-csslint": "0.2.0",
-    "gulp-filter": "3.0.1",
-    "gulp-util": "3.0.6",
+    "gulp-csslint": "1.0.0",
```
Comment (obuckley-kenzan) [inline package.json]: two of these are major upgrades, any concerns over backwards breaking changes?

**Comment by obuckley-kenzan** [comment]
ok to test

File: package.json (hunk 1 above)
Comment (paul-barry-kenzan) [inline package.json]: @cxoltero 
Do the docs for either of these libraries (`gulp-csslint` and `gulp-filter`) mention anything about breaking changes between the versions? Any mention of breaking changes in those docs should be confirmed in our code base, ideally with unit tests.

File: test/fixtures/test-css2.css [hunk 2]
```diff
@@ -1,5 +1,6 @@
 .somediv:hover {
   background-color: inherit;
   opacity: 0;
-  display: flex;
+  display: flex;;
```
Comment (paul-barry-kenzan) [inline test/fixtures/test-css2.css]: Extra `;`?

File: package.json (hunk 1 above)
Comment (cxoltero) [inline package.json]: @paul-barry-kenzan  | @obuckley-kenzan 
For gulp-csslint the major change was renaming 'reporter' to 'formatter'. they did this to match CSSlint. No breaking changes. for gulp-filter, the main change is a bug fix for relative path to match its current working directory. [gulp-filter relative path](https://github.com/sindresorhus/gulp-filter/commit/5fd40bcb63978b5d87468143ade5e7a56692c129). All unit test pass on this.

File: test/fixtures/test-css2.css (hunk 2 above)
Comment (thescientist13) [inline test/fixtures/test-css2.css]: @paul-barry-kenzan 
I think that fixture is used for failure testing?

Comment (paul-barry-kenzan) [inline test/fixtures/test-css2.css]: :man_facepalming: 
You're right. I guess the `fixtures` directory would be a dead give away, huh?

Comment (cxoltero) [inline test/fixtures/test-css2.css]: :grin:   ^^

**Comment by thescientist13** [comment]
resolves #20

**Pull request merged**